l10n_es_facturae/views/report_facturae.xml

```diff
@@ -317,7 +317,7 @@
                             <TransactionDate t-if="False"/>
                             <AdditionalLineItemInformation t-length="2500" t-if="False"/>
                             <SpecialTaxableEvent t-if="False"/>
-                            <ArticleCode t-length="20" t-if="False"/>
+                            <ArticleCode t-length="20" t-esc="line.product_id.default_code" t-if="line.product_id and line.product_id.default_code"/>
```

Comment:
No necesitas la doble condición. Directamente `t-if="line.product_id and line.product_id.default_code"`

Reply:
t-esc, no?

Reply:
t-esc es para poner ese valor, sí. Yo me refiero a la otra parte del t-if

Reply:
Ahora ya te entendí :wink: